create_database.py

```diff
@@ -0,0 +1,47 @@
+import database_auth
+import requests
+import json
+from mysql.connector import Error
+
+connection = database_auth.connection()
+
+api_url = 'https://api.tfl.gov.uk/bikepoint'
+
+
+try:
+	if connection.is_connected():
+		cursor = connection.cursor()
+		## create table in database
+		create_table_query = "create table bikeapitest(id varchar(30) NOT NULL, common_name varchar(70) NOT NULL, terminal_id int NOT NULL, bike int, empty_docks int, total_docks int,update_time varchar(20), update_date varchar(20), CONSTRAINT UC_bikeapi UNIQUE(id, terminal_id));"
```

Comment:
See if you can use an ORM like module like SQLAlchemy - it makes managing DB queries very easy.